README.md

```diff
@@ -3,31 +3,49 @@
 
 A python wrapper around libprio.
 
-This library provides low-level bindings to the reference C implementation of the [Prio system](https://github.com/mozilla/libprio) and a high-level Python interface.
+This library provides low-level bindings to the reference C implementation of
+the [Prio system](https://github.com/mozilla/libprio) and a high-level Python
+interface.
 
 
 ## Build
 
+The `libprio` submodule must be initialized before building. Run the following
+command to initialize the modules.
+
+```bash
+$ git submodule update --init --recursive
```

Comment:
Do you need the `--recursive`? Also, I prefer not to prefix base commands with `$` -- it makes copypasting more difficult and sometimes even throws newcomers to projects like this.

Reply:
There aren't any recursive sub-modules, so probably not. As for `$`, I've used it to distinguish between input and output -- but you're right in that it makes it hard to copy and paste. I'll clean up the readme a little more.